app/controllers/admin/routes.py

```diff
@@ -17,12 +17,16 @@
 from app.models.outsideParticipant import OutsideParticipant
 from app.models.eventParticipant import EventParticipant
 from app.models.programEvent import ProgramEvent
-
+from app.models.interest import Interest
+from app.models.programBan import ProgramBan
+from app.logic.getSLInstructorTableData import getProposalData
 from app.logic.participants import trainedParticipants
 from app.logic.volunteers import getEventLengthInHours
 from app.logic.utils import selectFutureTerms
-from app.logic.events import deleteEvent, getAllFacilitators, attemptSaveEvent, preprocessEventData, calculateRecurringEventFrequency
+from app.logic.events import deleteEvent, getAllFacilitators, getUpcomingEventsForUser , attemptSaveEvent, preprocessEventData, calculateRecurringEventFrequency
 from app.controllers.admin import admin_bp
+from app.logic.users import isEligibleForProgram
+from app.logic.searchUsers import searchUsers
```

Comment:
You added a lot of imports in this file, but there are no other changes. Why did you need the imports?

Reply:
Removed the unused imports